src/dc_mimefactory.rs

```diff
@@ -1034,9 +966,9 @@ pub unsafe fn dc_mimefactory_render(context: &Context, factory: &mut dc_mimefact
                 );
             }
             if e2ee_helper.encryption_successfull {
-                factory.out_encrypted = 1;
+                factory.out_encrypted = false;
```

Comment:
This should be `true` if it was `1` before

Reply:
Fixed.